lr_face/evaluators.py

```diff
@@ -260,6 +316,10 @@ def evaluate(experiment: Experiment,
             savefig=f'{make_plots_and_save_as} tippett.png'
         )
 
+        plot_cllr(
```

Comment:
this plot is only useful when there is enfsi selected as data, but is made anyway. Can we do anything to prevent that (or maybe make it clear in the function name? (plot_enfsi_cllr or something?)